src/System.Diagnostics.DiagnosticSource/tests/Configurations.props

```diff
@@ -1,8 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
-      netfx-Windows_NT;      
+      netcoreapp;
+      netfx-Windows_NT;
```

Comment:
I know it was there already but since you're doing this cleanup, the netfx configuration should be OS Agnostic.

Reply:
same comment as below.